validators/email_is_present.go

```diff
@@ -16,12 +17,37 @@ type EmailIsPresent struct {
 }
 
 // IsValid performs the validation based on the email regexp match.
-func (v *EmailIsPresent ) IsValid(errors *validate.Errors) {
+func (v *EmailIsPresent) IsValid(errors *validate.Errors) {
 	r := regexp.MustCompile(rxEmail)
 	if !r.Match([]byte(v.Field)) {
 		if v.Message == "" {
 			v.Message = fmt.Sprintf("%s does not match the email format.", v.Name)
 		}
 		errors.Add(GenerateKey(v.Name), v.Message)
 	}
-}
\ No newline at end of file
+}
+
+type EmailLike struct {
```

Comment:
Can you please comment this so people know the difference between `EmailIsPresent` and `EmailLike`?